docs/tutorials/qGAN.ipynb

```diff
@@ -0,0 +1,2644 @@
+{
```

Comment:
<p>If it was learning correctly these graphs would show it tending towards a lognormal distribution. However, as you can see it isn't learning correctly. I have tried fiddling around with the learning rate etc. but I can't seem to fix it. Also, I don't think the entropy calculation is working correctly but it isn't necessary - I might just remove it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/quantum/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>